tests/unit/type/hash.tcl

```diff
@@ -1,4 +1,18 @@
 start_server {tags {"hash"}} {
+    proc hash_response_interpreter {id response} {
+        if {!$::force_resp3 || $::redis::testing_resp3($id) == 1} {
+            return $response
+        }
+        if {[string is list $response] && [string is list [lindex $response 0]] && [llength [lindex $response 0]] eq 2} {
+            set flatarray {}
+            foreach pair $response {
+                lappend flatarray {*}$pair
+            }
+            return $flatarray
+        }
+        return $response
+    }
```

Comment:
missing some comment about what it does and for which commands.
i.e. flattern output of HRANDFIELD